### PR TITLE
Add Section 1 to Step 4 document (ENG ver.)

### PR DESCRIPTION
**Pull request by e-yang08**
## What
- Added the Section 1 to Step 4 document to make it the same as Japanese version
Note: I replaced the reference link with the english version in the Docker official documentation.

## CHECKS :warning:
Please make sure you are aware of the following.

- [x] **The changes in this PR doesn't have private information

### CODE REVIEW

File: document/step4.en.md [hunk 1]
```diff
@@ -76,7 +84,7 @@ Check that you can now see `build2022/app` in the list of images.
 
 * [Dockerfile reference](https://docs.docker.com/engine/reference/builder/)
 
-## 4. Modity Dockerfile
+## 5. Modity Dockerfile
 **Run the docker image you built in STEP4-3, and check if the following error shows up.**
```
Comment (wakanapo) [inline document/step4.en.md:88]:
```suggestion
**Run the docker image you built in STEP4-4, and check if the following error shows up.**
```

File: document/step4.en.md [hunk 2]
```diff
@@ -96,7 +104,7 @@ Run the image with the modified `dockerfile`, check if the same message is displ
 * [docker docs - language guide (Python)](https://docs.docker.com/language/python/)
 * [docker docs - language guide (Go)](https://docs.docker.com/language/golang/)
 
-## 5. Run the listing API on Docker
+## 6. Run the listing API on Docker
 
 The environment within the docker image should be the same as STEP2-2 after STEP4-4.
```
Comment (wakanapo) [inline document/step4.en.md:109]:
```suggestion
The environment within the docker image should be the same as STEP2-2 after STEP4-5.
```

**Review by wakanapo — COMMENTED** [review]
Thank you for your contribution!
I added some suggestions.

File: document/step4.en.md (hunk 1 above)
Comment (e-yang08) [reply document/step4.en.md:88]: Thank you! Sorry I didn't pick up that.

**Comment by e-yang08** [comment]
> Thank you for your contribution! I added some suggestions.

Thank you very much for double-checking. Really appreciated it.

**Review by wakanapo — APPROVED** [review]
LGTM 👍 
Thank you.

**Pull request merged**